SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -2,15 +2,17 @@
 # Get started 
 ## 2 - Get and build the SpatialOS Unreal Engine Fork
 
-To use the SpatialOS GDK for Unreal, you first need to build the SpatialOS fork of Unreal Engine.
+To use the SpatialOS GDK for Unreal, you need to get the SpatialOS-compatible version of Unreal Engine - this is the SpatialOS Unreal Engine Fork. You get it as source code from GitHub and then build it.
 
 ### Step 1: Unreal Engine EULA
 
-To get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement (EULA) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You cannot use the GDK without doing this first. To do this, see the [Unreal Engine documentation](https://www.unrealengine.com/en-US/ue4-on-github).
+To get access to the SpatialOS fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement (EULA) and accept the invite to join the [EpicGames organisation on GitHub](https://github.com/EpicGames). You cannot use the GDK without doing this first. </br>
+To do this, see the [Unreal Engine documentation](https://www.unrealengine.com/en-US/ue4-on-github).
 
-### Step 2: Get the Unreal Engine fork source code and Unreal Linux cross-platform support
+### Step 2: Get the Unreal Engine Fork source code and Unreal Linux cross-platform support
 
-1.  Open a terminal and run either of these commands to clone the [Unreal Engine fork](https://github.com/improbableio/UnrealEngine) repository.
+1. **Unreal Engine Fork**</br> 
+Open a terminal and run either of these commands to clone the [Unreal Engine fork](https://github.com/improbableio/UnrealEngine) repository.
 
     > **TIP:** Clone the Unreal Engine fork into your root directory to avoid file path length errors. For example: `C:\GitHub\UnrealEngine`. 
```

Comment:
```suggestion
    > **TIP:** Clone the Unreal Engine Fork into your root directory to avoid file path length errors. For example: `C:\GitHub\UnrealEngine`. 
```
(There might be some others that aren't in the bit I can comment on - I haven't checked, so it's worth doing a quick sweep)